test/integration/targets/ns1_record/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/cloud/group1
```

Comment:
Use `unsupported` here instead, since these tests cannot run on Shippable.

Reply:
Makes sense, thanks.  Should be all set.